src/molecules/ProjectHeader.jsx

```diff
@@ -9,7 +9,7 @@ export default function ProjectHeader({
   title,
 }) {
   const projectHomeUrl = `/projects/${id}`;
-  const projectSettingsUrl = `/projects/${id}/settings`;
+  const projectSettingsUrl = projectHomeUrl + "/settings";
```

Comment:
Was `projectHomeUrl` going to be used elsewhere?

Reply:
It's also used on line 20 of the same file.